.github/PR_TEMPLATE/PULL_REQUEST_CP_TEMPLATE.md

```diff
@@ -1,73 +1,50 @@
-**This pull request is opened against a release branch.<br>
-To request a cherry pick of a commit, please fill in the form below.** (Questions with an asterisk are required.)<br>
+This pull request is created by [automatic cherry pick workflow](https://github.com/flutter/flutter/wiki/Flutter-Cherrypick-Process#automatically-creates-a-cherry-pick-request)
+Please fill in the form below, and a flutter domain expert will evaluate this cherry pick request.
 
-**To fill in the form, you can edit this PR description and type your answers after the 'My Answer' keywords. <br>
-A flutter domain expert will evaluate this cherry pick request shortly after ALL questions are answered.**
+---
```

Comment:
We don't need dividers, when using an H2(##) dividers are built in.

However, we may want to remove them completely and change the headings to H3(###) to still provide visual space without adding clutter.

Reply:
Good point, changed to H3 and removed dividers